src/support/htable.inc

```diff
@@ -9,8 +9,8 @@
 // compute empirical max-probe for a given size
 #define max_probe(size) ((size)<=(HT_N_INLINE*2) ? (HT_N_INLINE/2) : (size)>>3)
 
-#define _HTIMPL_EX(HTNAME, HFUNC, EQFUNC, _STATIC)                      \
-static void **HTNAME##_lookup_bp_r(htable_t *h, void *key, void *ctx)   \
+#define _HTIMPL_EX(HTNAME, HFUNC, EQFUNC, _STATIC, _SAFE)               \
```

Comment:
These annotations usually belong on the declarations (in the header), not the definitions. See the existing ones here for an example: https://github.com/JuliaLang/julia/blob/b961066c838fbb4c5415b5e97e520843f58e3376/src/support/htable.h#L43